### PR TITLE
IK3I's Fluff Webbing

### PR DESCRIPTION
**Pull request by IK3I**
Adds Patreon Fluff Item for IK3I

Still need a backend SQL edit to actually spawn with it, but this takes care of the code and sprite side of it.

Character - Yakikatachi

Spritesheets:
Ground and Inv
Imgur

Worn (off)
Imgur

Worn (on)
Imgur

Demo gifs:
Imgur

Imgur

Imgur

96 X 96 stills
Imgur
Imgur
Imgur
Imgur

### CODE REVIEW

**Comment by taukausanake** [comment]
_Squints eyes_
THESE ARE SO TINY

**Comment by IK3I** [comment]
It's 100% scaling, in retrospect, probably should've zoomed it in during editing because turns out 10-20 pixels isn't very big on any monitor

**Comment by KasparoVy** [comment]
96x96 might be the way to go for previews.

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -649,6 +673,32 @@
...
+/obj/item/clothing/suit/storage/fluff/k3_webbing
+	name = "vox tactical webbing"
+	desc = "A somewhat worn but well kept set of vox tactical webbing. It has a couple of pouches attached."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "k3_webbing_off"
+	species_fit = list("Vox")
+	sprite_sheets = list("Vox" = 'icons/mob/species/vox/suit.dmi')
+	var/mode = 0
+
+/obj/item/clothing/suit/storage/fluff/k3_webbing/verb/toggle()
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: this should be an action button, not an ugly verb.

Comment (IK3I) [inline code/modules/customitems/item_defines.dm]: Haven't touched action buttons but I'll find an example and remake it to fit

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -649,6 +673,37 @@
...
+		else
+			flavour = "on"
+			icon_state += "_on"
+			item_state += "_on"
+			suit_adjusted = 1 //Lights On
+
+		for(var/X in actions)
+			var/datum/action/A = X
+			A.UpdateButtonIcon()
+		to_chat(user, "You turn the lighting system on \the [src] [flavour].")
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: no `\the`.

**Comment by tigercat2000** [comment]
@IK3I This can be merged as soon as the conflicts are resolved.

**Comment by IK3I** [comment]
oh god that did not merge right oops

**Pull request merged**